src/__tests__/core/account.test.ts

```diff
@@ -45,3 +44,46 @@ test('account: select account balance', async () => {
   expect(result.errors).toBeUndefined();
   expect(result.data.account.balance).toBeGreaterThan(0);
 });
+
+test('account: select account balance with unit conversion', async () => {
+  const units =  {'wei': 1,
```

Comment:
Wondering if this conversion table is available in the web3 library? Do we need to re-create it manually in the test?